src/main/resources/story/shop_api/PostCustomers.story

```diff
@@ -0,0 +1,16 @@
+Story: POST /cusomers/ test
+
+Scenario: check response body
+When I send HTTP POST to the relative URL '/customers/' with content: '{
```

Comment:
Is there step for sending HTTP POST to the absolute URL?
I added property `web-application.main-page-url=https://api.predic8.de:443/shop` to environment.properties but this step still fails

Reply:
1. Example of absolute url usage: When I issue a HTTP HEAD request for a resource with the URL 'https://google.com/'
2. To use steps with parameter 'relative url' you need to open main-page-url. As example:
```
Given I am on the main application page

```

Reply:
property `web-application.main-page-url` refers to web UI actions.
for API I set up `rest-api.http.endpoint=https://api.predic8.de/shop/`